scripts/readelf.py

```diff
@@ -95,6 +96,11 @@ def _get_cu_base(cu):
     else:
         raise ValueError("Can't find the base IP (low_pc) for a CU")
 
+_CONTROL_CHAR_RE = re.compile(r'[\x01-\x1f]')
```

Comment:
Please add a comment explaining what this is for

Reply:
I've added an explanation for it (meant for use in `_format_symbol_name`).